spec/lucky/error_handling_spec.cr

```diff
@@ -2,80 +2,137 @@ require "../spec_helper"
 
 include ContextHelper
 
-private class FakeError < Exception
+private class CustomError < Exception
+end
+
+private class StealthError < Exception
 end
 
 private class UnhandledError < Exception
 end
 
-private class ShortAndStoutError < Exception
-  include Lucky::HttpRespondable
+# Can't be private because if it is Crystal won't let us use temp_config
+class FakeErrorAction < Lucky::ErrorAction
+  default_format :html
+  dont_report [StealthError]
 
-  def http_error_code : Int32
-    418
+  Habitat.create do
+    setting output : IO = IO::Memory.new
   end
-end
 
-private class InvalidParam < Lucky::Exceptions::InvalidParam
-end
-
-private class FakeErrorAction < Lucky::ErrorAction
-  def handle_error(error : FakeError) : Lucky::Response
+  def render(error : CustomError) : Lucky::Response
```

Comment:
Renamed `handle_error` to `render`. I think this is simpler and reads better

Reply:
Does this cause any issues with calling the `render` method from actions? I guess it shouldn't if their argument types are different.

Reply:
I think it will work (or it wouldn't compile) but this could be kind of confusing or lead to unexpected compile errors since there could be tons of different overloads. I think for now I'll rename it  `render_error`

I've been thinking of renaming `render` to `html` or something. Since everything else (like `json`) has the type in the name. E.g. `html Users::IndexPage, users: UserQuery.new`

Reply:
I like the idea of using `html`. It's an easy change.

Reply:
Done! I like it. And should be a pretty easy thing to upgrade